vnext/ReactUWP/Views/DatePickerViewManager.cpp

```diff
@@ -58,10 +59,12 @@ void DatePickerShadowNode::createView() {
 
 void DatePickerShadowNode::updateProperties(const folly::dynamic &&props) {
   m_updating = true;
+  const auto datePicker =
+      XamlDirectInstance::GetXamlDirect().GetXamlDirectObject(
```

Comment:
Did you do the test on old OS? IXamlDirectObject  is introduced in 1809, but I didn't see any fallback logic in your code. and I don't know if you introduce a bug.

Reply:
Yes, thank you for pointing this out. I'm planning to use this nuget package to work on down-level.
https://www.nuget.org/packages/Microsoft.UI.Xaml

The source is here 
https://microsoft.visualstudio.com/OS/_git/dep.xamldirectlib?path=%2Fsrc%2FMicrosoft.UI.Xaml.Core.Direct%2FXamlDirectWrapper.cpp&version=GBmaster

On down-level OS versions it just performs the equivalent operation using the available apis. on 1809+ it is just a pass through.

One thing that's interesting. The binaries in the nuget are ~2MB which is annoying for apps that target a min os with IXamlDirect. I'm not sure if it will be easy to conditionally link against the nuget based on min os, but to me that seems ideal until xaml lift is complete.